Makefile

```diff
@@ -518,78 +311,7 @@ init-conda:
 
 #<<<
 .PHONY: init-docker
-init-docker: ssl-cert docker-networks docker-volumes docker-secrets
-
-
-#>>>
-# initialize kubernetes environment
-# make init-kubernetes
-
-#<<<
-.PHONY: init-kubernetes
-init-kubernetes: ssl-cert docker-secrets docker-pull
-	$(DIR)/bin/kubectl create namespace $(DOCKER_NAMESPACE)
-
-
-#>>>
-# initialize docker-swarm environment and create swarm networks, configs, and secrets
-# make init-swarm
-
-#<<<
-.PHONY: init-swarm
-init-swarm: swarm-init swarm-networks swarm-configs swarm-secrets
-
-
-#>>>
-# deploy/test all modules in $CANDIG_MODULES using Kubernetes
-# make kubernetes
-
-#<<<
-.PHONY: kubernetes
-kubernetes:
-	$(DIR)/bin/kompose --file $(DIR)/lib/kubernetes/docker-compose.yml \
-		$(foreach MODULE, $(CANDIG_MODULES), --file $(DIR)/lib/$(MODULE)/docker-compose.yml) \
-		up
-
-
-#>>>
-# deploys individual module using kompose
-# $module is the name of the sub-folder in lib
-# make kube-$module
-
-#<<<
-kube-%:
-	$(DIR)/bin/kompose --file $(DIR)/lib/kubernetes/docker-compose.yml \
-		--file $(DIR)/lib/$*/docker-compose.yml up
-
-
-#>>>
-# create docker-machine instance(s) for Docker Compose/Swarm development
-# NOTE: only virtualbox is supported at this time
-# NOTE: use MINIKUBE_* to configure vm options
-# $vm_name must be a unique name for the docker-machine instance (e.g. make machine-manager)
-# make machine-$vm_name
-
-#<<<
-machine-%:
-	$(DIR)/bin/docker-machine create --driver "$(MINIKUBE_DRIVER)" \
-  		--virtualbox-cpu-count "$(MINIKUBE_CPUS)" --virtualbox-memory "$(MINIKUBE_MEM)" \
-		--virtualbox-disk-size "$(MINIKUBE_DISK)" --virtualbox-hostonly-cidr "192.168.56.1/24" \
-		--virtualbox-hostonly-nicpromisc "deny" --virtualbox-hostonly-nictype "82540EM" \
-		$*
-
-
-#>>>
-# create minikube environment for (kubernetes) integration testing
-# make minikube
-
-#<<<
-.PHONY: minikube
-minikube:
-	$(DIR)/bin/minikube start --container-runtime $(MINIKUBE_CRI) \
-		--cpus $(MINIKUBE_CPUS) --memory $(MINIKUBE_MEM) --disk-size $(MINIKUBE_DISK) \
-		--network-plugin cni --cni $(MINIKUBE_CNI) --driver $(MINIKUBE_DRIVER) \
-		--dns-domain $(CANDIG_DOMAIN) --nodes $(MINIKUBE_NODES)
+init-docker: docker-volumes docker-secrets
```

Comment:
Why are we taking out ssl-cert?

Reply:
We no longer need self-signed certs. In the cases where we do use SSL, it is handled by an external ingress/proxy.